tests/test_github_sdk.py

```diff
@@ -120,7 +120,7 @@ def test_send_trace(
         "Content-Type": "application/x-sentry-envelope",
         "Content-Encoding": "gzip",
         "X-Sentry-Auth": f"Sentry sentry_key=foo,sentry_client=gha-sentry/0.0.1,sentry_timestamp={now},sentry_version=7",
-        "Content-Length": "701",
+        "Content-Length": "693",
```

Comment:
better gzip?

Reply:
Possibly the trace ID is a little shorter.